x/concentrated-liquidity/fees_test.go

```diff
@@ -699,3 +699,114 @@ func (s *KeeperTestSuite) TestCollectFees() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestUpdateFeeAccumulatorPosition() {
+	var (
+		ownerOne = s.TestAccs[0]
+		ownerTwo = s.TestAccs[1]
+	)
+
+	type updateFeeAccumPositionTest struct {
+		poolId        uint64
+		owner         sdk.AccAddress
+		liquidity     sdk.Dec
+		lowerTick     int64
+		upperTick     int64
+		expectedError error
+	}
+
+	positions := map[int]updateFeeAccumPositionTest{
+		1: {
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick,
+		},
+		2: {
+			lowerTick: DefaultLowerTick + 1,
+			upperTick: DefaultUpperTick,
+		},
+		3: {
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick + 1,
+		},
+	}
+
+	tests := map[string]updateFeeAccumPositionTest{
+		"update position with a different pool ID": {
+			poolId:    2,
+			owner:     ownerOne,
+			liquidity: DefaultLiquidityAmt,
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick,
+		},
+		"update position with a different owner": {
+			poolId:    1,
+			owner:     ownerTwo,
+			liquidity: DefaultLiquidityAmt,
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick,
+		},
+		"update position with a different lower tick": {
+			poolId:    1,
+			owner:     ownerOne,
+			liquidity: DefaultLiquidityAmt,
+			lowerTick: DefaultLowerTick + 1,
+			upperTick: DefaultUpperTick,
+		},
+		"update position with a different upper tick": {
+			poolId:    1,
+			owner:     ownerOne,
+			liquidity: DefaultLiquidityAmt,
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick + 1,
+		},
+	}
+
+	for name, tc := range tests {
+		s.Run(name, func() {
+			s.SetupTest()
+
+			// Setup two cl pools
+			poolOne := s.PrepareConcentratedPool()
+			poolTwo := s.PrepareConcentratedPool()
+
+			pools := []cltypes.ConcentratedPoolExtension{poolOne, poolTwo}
+			owners := []sdk.AccAddress{ownerOne, ownerTwo}
+
+			// Initialize three base positions in each pool for each owner (total of 12 positions)
+			for _, pos := range positions {
```

Comment:
We could also just initialize this once and for all the test cases

Reply:
Same point as above